### DISCUSSION TITLE
I made a Linux FlatPak Tutorial, showing how to install it on a Flatpak OBS.

### DISCUSSION THREAD

**Discussion by AntonioGomes42** [sorayuki/obs-multi-rtmp#282] [Show and tell]
For helping some users, I made a tutorial of how to install this OBS plugin on linux, in a Flatpak OBS, it works for every single linux. Link : https://youtu.be/p4eoEoq438M. I hope it helps you!

**Comment by sxnvmqe** [comment]
https://github.com/sxnvmqe/obs-multi-rtmp-flatpak
